content/project/leon.md

```diff
@@ -0,0 +1,19 @@
+---
+start_date: 2016
+end_date: 2017
```

Comment:
Nie do końca jestem pewien tego, czy to są rzeczywiste daty rozpoczęcia i zakończenia projektu